server/routes/visitJourney/dateAndTime.ts

```diff
@@ -15,19 +15,33 @@ export default class DateAndTime {
   ) {}
 
   async get(req: Request, res: Response): Promise<void> {
+    let daysUntilVisitStart = 2
+
     const isUpdate = this.mode === 'update'
     const { prisonId } = req.session.selectedEstablishment
     const { visitSessionData } = req.session
+
```

Comment:
If you use something like this:
```
const warningMessages: { id: string; message: string }[] = []
```
then you have a (default empty) array of warning messsages that you can use with a `for` loop in the template and you'll have the message and id / test values that are necessary